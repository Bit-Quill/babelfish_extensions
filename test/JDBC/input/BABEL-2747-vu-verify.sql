SELECT type, type_desc from sys.procedures  where name like 'babel_2747__vu_prepare%' order by type;
GO

SELECT type, type_desc from sys.all_objects  where name like 'babel_2747__vu_prepare%' order by type;
GO
