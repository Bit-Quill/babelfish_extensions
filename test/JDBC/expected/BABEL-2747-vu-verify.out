SELECT type, type_desc from sys.procedures  where name like 'babel_2747__vu_prepare%' order by type;
GO
~~START~~
char#!#nvarchar
FN#!#SQL_SCALAR_FUNCTION
P #!#SQL_STORED_PROCEDURE
TR#!#SQL_TRIGGER
~~END~~


SELECT type, type_desc from sys.all_objects  where name like 'babel_2747__vu_prepare%' order by type;
GO
~~START~~
char#!#nvarchar
IF#!#SQL_INLINE_TABLE_VALUED_FUNCTION
P #!#SQL_STORED_PROCEDURE
TR#!#SQL_TRIGGER
U #!#USER_TABLE
~~END~~

